### ISSUE TITLE
Metric for evaluating cluster?

### ISSUE THREAD

**Issue by liamcarroll** [Crone1/CA4015#1]
Well lads, I found a metric for evaluating clustering algorithms that might be handy to use called the Silhouette Coefficient. It has an implementation in scikit-learn, here's the link: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.silhouette_score.html.

Let me know what's yous think.

**Comment by weldonj** [comment]
Nice one, will read up on it now and we can discuss on next call

**Comment by Crone1** [comment]
Looks good, great idea. We can look into this on our call today
